terraform/groups/configurable-api-caller/main.tf

```diff
@@ -140,6 +140,30 @@ resource "aws_lambda_permission" "allow_cloudwatch_dissolutions_rebel1" {
   source_arn    = aws_cloudwatch_event_rule.call_api_caller_lambda_dissolutions_rebel1[0].arn
 }
 
+resource "aws_cloudwatch_event_rule" "call_api_caller_lambda_efs_handle_delayed_submission_sameday" {
+  count               = var.deploy_to == "development" ? 1 : 0
+  name                = "call_api_caller_lambda_efs_handle_delayed_submission_sameday"
+  description         = "Cloudwatch event to call ${aws_lambda_function.configurable_api_lambda.function_name} lambda routinely"
```

Comment:
We should be updating this description to say why we're calling the api caller.

Reply:
added to description, is this what you had in mind? / sufficient ?